### ISSUE TITLE
Unit test view is hard to navigate on Windows

### ISSUE THREAD

**Issue by lesquoyb** [gama-platform/gama#159]
**Describe the bug**
Scrolling in the `Tests` view is extremely annoying as (on windows) you have to scroll way more than in the other views, to the user it feels like the view is lagging (it is not, it just requires two times more effort than usual to scroll but everything works properly)

**To Reproduce**
Run all the unit tests and once it's done try to scroll to the last test

**Expected behavior**
It is as easy to scroll as in other views

**Comment by AlexisDrogoul** [comment]
I can't reproduce the same "feeling" on macOS. It's probably specific to the way Windows treats "virtual" list views on SWT...

**Comment by GuillaumeLeroy34** [comment]
I have also experienced the bug

**Comment by lesquoyb** [comment]
Also want to add that it is the same in the settings window

**Comment by AlexisDrogoul** [comment]
Is it the same in the #690 PR ?

**Comment by lesquoyb** [comment]
yes, no difference for both the test view and the preferences window

**Comment by lesquoyb** [comment]
solved for the unit test view but now I'm completely unable to open the settings window, it never appears and gama is like frozen: I cannot interact with it anymore even though the process is not blocked by Windows so it feels like there's something "above" the gama program that is catching everything. I can't even move or resize gama with the Windows keyboard shortcuts

**Comment by lesquoyb** [comment]
Tried on main branch and latest alpha, it happens with and without external monitor. If I force quit with the task manager I have this:

![Image](https://github.com/user-attachments/assets/716d39aa-9fa4-4ed4-ab8d-743179db0a2a)

**Comment by lesquoyb** [comment]
Also other modals load without any problem.
In the eclipse console I have this line after clicking on preferences:
```
> GAMA  : Preloading preferences view                   done in_______ 1ms
```

**Comment by AlexisDrogoul** [comment]
Can you check in the registry the value of the 3 first prefs ? 

<img width="257" alt="Image" src="https://github.com/user-attachments/assets/622114de-4825-442f-977f-cc2bb130b4c7" />

**Comment by AlexisDrogoul** [comment]
On my side no problem at all .... :( 

![Image](https://github.com/user-attachments/assets/6145ce2b-6400-4e63-8272-e07248ec82c1)

**Comment by lesquoyb** [comment]
Ah yes, somehow it was saved outside of my monitors. Navigation is fixed in the preferences too

**Comment by AlexisDrogoul** [comment]
Actually, this is a bug (already described here : #492 ) -- maybe we should fix it :)
